src/main/java/com/dev/bootbasic/CommandLineApplication.java

```diff
@@ -52,7 +58,11 @@ private void createVoucher() {
 
     private void showAllVouchers() {
         List<VoucherDetailsResponse> allVouchers = voucherController.findAllVouchers();
-        viewManager.showCollectionMessage(allVouchers);
+
+        List<VoucherDetailsViewResponse> voucherResponses = allVouchers.stream()
+                .map(VoucherDetailsViewResponse::from)
+                .collect(toUnmodifiableList());
```

Comment:
```suggestion
                .toList();
```
요렇게 호출하면 `unmodifiableList`로 반환할꺼에요

Reply:
감사합니다 반영했습니다!
[74e9cc7](https://github.com/prgrms-be-devcourse/springboot-basic/pull/756/commits/74e9cc7bf14fd97e895ea4ab0a74230e6678e1ed)